src/otx/core/exporter/base.py

```diff
@@ -150,25 +163,45 @@ def to_exportable_code(
             output_dir (Path): path to the directory to store export artifacts
             base_model_name (str, optional): exported model name
             precision (OTXExportPrecisionType, optional): precision of the exported model's weights
+            path_to_already_exported_model (PathLike, optional): path to the already
+                exported model to add it in exportable code
 
         Returns:
             Path: path to the exported model.
         """
-        work_dir = Path(demo.__file__).parent
+        work_demo_dir = Path(demo.__file__).parent
+        training_extensions_dir = Path(otx.__file__).parents[2]
```

Comment:
I think it doesn't work if otx is installed as not editable mode. Could you copy LICENSE to demo directory and use it? You can refer #3255

Reply:
Ok, let's add LICENSE to demo folder